face-dection-models/dnn.py

```diff
@@ -0,0 +1,34 @@
+import cv2
+
+
+def detectFaceOpenCVDnn(net, frame, framework="caffe", conf_threshold=0.7, detectMultipleFaces=False):
+    frameOpencvDnn = frame.copy()
+    frameHeight = frameOpencvDnn.shape[0]
```

Comment:
have `img` as input for the functions. Also make them have the same style of name: `detect_face_dnn` etc